### ISSUE TITLE
InvalidConsoleArgumentValue raised with the use a ternary operator

### ISSUE THREAD

**Issue by gillesdalle** [psalm/psalm-plugin-symfony#252]
Hello,

With psalm v3.1.4, the error **InvalidConsoleArgumentValue** is raised with this code, but is's a false positive :
```
new InputArgument(
    'env',
    null === $this->defaultEnv ? InputArgument::REQUIRED : InputArgument::OPTIONAL,
    'The application environment to dump .env files for - e.g. "prod".'
);
```
```sh
ERROR: InvalidConsoleArgumentValue - src/Command/DumpEnvCommand.php:50:21 - Use Symfony\Component\Console\Input\InputArgument constants
                    null === $this->defaultEnv ? InputArgument::REQUIRED : InputArgument::OPTIONAL,
```
